skyportal/models.py

```diff
@@ -585,6 +585,14 @@ class Obj(Base, ha.Point):
         doc="Notifications regarding the object sent out by users",
     )
 
+    listings = relationship(
+        "Listing",
+        back_populates="obj",
+        cascade="delete",
```

Comment:
cc @acrellin  should there be more cascade args ?

Reply:
Yeah this is one of the things I wasn't sure about...

Reply:
```suggestion
        cascade="save-update, merge, refresh-expire, expunge",
```